### ISSUE TITLE
Does csx have plans to support calc() ?

### ISSUE THREAD

**Issue by minodisk** [typestyle/csx#32]
Like this:

1\.
```typescript
style({
  width: calc(`${percent(50)} - ${px(100)}`),
})
```

or

2\.
```typescript
style({
  width: calc(percent(50) - px(100)),
})
```

2 is probably impossible. Because the JavaScript runtime will calculate.

**Comment by notoriousb1t** [comment]
As you said, 2 is impossible because you can't overload JavaScript operators, but there should be no issue adding calc() as in 1.

**Comment by notoriousb1t** [comment]
This has been released with v8.3.0 along with some other updates

**Comment by minodisk** [comment]
Thanks for your replying.
I'm trying v8.3.0 and `calc(exp: string): string` seems to work :sushi:
